source/transactions/transactions.rst

```diff
@@ -810,26 +810,30 @@ certain retries of commitTransaction and abortTransaction) to the same mongos.
 When to unpin
 ^^^^^^^^^^^^^
 
-Drivers MUST unpin a ClientSession when a command within a transaction,
-including commitTransaction and abortTransaction, fails with a
-TransientTransactionError. Transient errors indicate that the transaction
-in question has already been aborted or that the pinned mongos is
+Drivers MUST unpin a ClientSession in the following situations:
+
+#. The ``abortTransaction`` command is executed. The session MUST be unpinned
```

Comment:
This is the only new criteria for unpinning that was added in this PR. All of the other changes in this file are to restructure the existing criteria in list form.